Userland/Applications/NetworkSettings/CMakeLists.txt

```diff
@@ -5,15 +5,18 @@ serenity_component(
 )
 
 compile_gml(NetworkSettings.gml NetworkSettingsGML.h network_settings_gml)
+compile_gml(CertStoreDialog.gml CertStoreDialogGML.h cert_store_dialog_gml)
 
 set(SOURCES
     NetworkSettingsWidget.cpp
+    CertStoreDialog.cpp
```

Comment:
not really sure if it should go exactly under network settings… itʼs more of the system and/or user setting instead.

while iʼd appreciate some kind of a link or dialog opener there, the certificates need not to be strictly related to the web or network…?

or we should be just explicit about what the certificates are related to exactly (then iʼd be more of browser than network thing? 🤔)

Reply:
I might actually scrap this PR. The current local state I'm adding features to is not part of NetworkSettings.

Reply:
well… moving that somewhere would be better instead! (but i do not now really where this would belong to the best!)

anyone? 😅

Reply:
We discussed that in Discord and it will be an independent app for now.